tests/test_popularity.py

```diff
@@ -27,9 +27,8 @@ def test_2arm_equal_prob(self):
                                 num_run=1,
                                 is_predict=False)
 
-        # Check that normalized probabilities are 1/2 each, and sum up to 1.0
```

Comment:
These aren't normalized now? Any reason why you wouldn't want to normalize the result of the dirichlet values?

Reply:
They are already normalized (sum = 1).
https://numpy.org/doc/stable/reference/random/generated/numpy.random.dirichlet.html